app/src/componentLibrary/radioGroup/radioGroup.jsx

```diff
@@ -31,7 +31,7 @@ export const RadioGroup = ({ options, value, ...rest }) => (
 RadioGroup.propTypes = {
   variant: PropTypes.string,
   options: PropTypes.array,
-  value: PropTypes.bool,
+  value: PropTypes.string,
```

Comment:
I suggest to use the next set of possible values:
```javascript
value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool]),
```

Refer props of [existing](https://github.com/reportportal/service-ui/blob/develop/app/src/components/inputs/inputRadio/inputRadio.jsx#L75) radio button